src/pages/index.jsx

```diff
@@ -18,6 +20,8 @@ const IndexPage = () => {
       <Hero />
       <ParaQueServe />
       <ComoUsar />
+      <PorqueOFazemos />
+      <LandingQuemSomos />
       <Footer />
```

Comment:
`Footer` should be gray in the landing page. My suggestion would be can add a `color` new prop to set background to gray or white.